Assets/GhostCollider.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GhostCollider : MonoBehaviour
+{
+    private CharacterController ghostController;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        ghostController = GetComponentInChildren<CharacterController>();
```

Comment:
is there even a characterController on the ghost prefab? there shouldn't be. just a capsule collider needs to be added

Reply:
The Unity help sites I was checking seemed sure that I needed a Rigidbody in order for triggers to function correctly. And charactercontroller was the easiest way to do that. I initially was working without one but couldn't get my triggers to go off (granted they still aren't working, but I'm trying to figure out what's happening).